sections/03_categorize.md

```diff
@@ -282,36 +281,42 @@ learning in the domain. Though not a methodological consideration, efforts are
 underway to recognize those who promote an ecosystem of rigorous sharing and
 analysis [@doi:10.1038/ng.3830].
 
-We expect both improved methods and an improved culture of sharing to
-play an important role if deep learning is going to transform how we analyze
-data to categorize states of human health. We don't expect that deep learning
-methods will replace expert review. We expect them to complement expert review
-by allowing more efficient use of the costly practice of manual annotation.
+We expect both improved methods and an improved culture of sharing to play an
+important role if deep learning is going to transform how we analyze data to
+categorize states of human health. We don't expect that deep learning methods
+will replace expert review. We expect them to complement expert review by
+allowing more efficient use of the costly practice of manual annotation.
 
 ###### Data sharing is hampered by standardization and privacy considerations
 
```

Comment:
Can you add this paragraph back?

300 To construct the types of very large datasets that deep learning methods thrive
301	on, we need robust sharing of large collections of data. This is in part a
302	cultural challenge. We touch on this challenge in the Discussion section. Beyond
303	the cultural hurdles around data sharing, there are also technological hurdles
304	related to sharing individual health records or deep models built from such
305	records. This subsection deals primarily with these challenges.

Can be obtained from:
https://github.com/greenelab/deep-review/blame/master/sections/03_categorize.md lines 300-305